src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -2716,9 +2716,21 @@ class FetchClauseSegment(BaseSegment):
             optional=True,
         ),
         OneOf("ROW", "ROWS"),
-        "ONLY",
+        OneOf("ONLY", Sequence("WITH", "TIES"))
```

Comment:
According to https://jakewheat.github.io/sql-overview/sql-2011-foundation-grammar.html#fetch-first-clause